shared/src/vyxal/Interpreter.scala

```diff
@@ -100,11 +100,10 @@ object Interpreter:
       case AST.CompressedString(value, _) =>
         ctx.push(StringHelpers.decompress252String(value))
       case AST.Lst(elems, _) =>
-        val list = collection.mutable.ListBuffer.empty[VAny]
-        for elem <- elems do
-          execute(elem)(using ctx.makeChild())
-          list += ctx.pop()
-        ctx.push(VList.from(list.toList))
+        val context = ctx.copy
+        context.clear()
```

Comment:
Maybe we could add a `stack` parameter to `Context#copy` to avoid having to clear later?